### PR TITLE
Template

### PR DESCRIPTION
**Pull request by willoci**

### CODE REVIEW

File: send-surveys/src/main/java/template/mgr/SurveyTemplateManager.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    public SurveyTemplateManager(MustacheFactory factory) {
+        this.factory = factory;
+    }
+
+    public Mustache getMustache(String mustacheFileName) {
+
+        return factory.compile(mustacheFileName);
+    }
+
+    public String populateTemplate(String templateFileName, Map<String, Object> data)
```
Comment (mkimberlin) [inline send-surveys/src/main/java/template/mgr/SurveyTemplateManager.java]: Make this second param  an Object

File: send-surveys/build.gradle [hunk 2]
```diff
@@ -32,7 +32,9 @@ dependencies {
     implementation "io.micronaut:micronaut-function-aws"
     implementation "javax.annotation:javax.annotation-api"
     runtimeOnly "com.amazonaws:aws-lambda-java-log4j2:1.0.0"
+    compile group: 'com.github.spullara.mustache.java', name: 'compiler', version: '0.9.6'
```
Comment (mkimberlin) [inline send-surveys/build.gradle]: Make this match style

File: send-surveys/src/main/resources/emailTemplate.mustache [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        </div>
+
+        <div class="wrapper">
+          <div class="question">How is your work day?</div>
+          <div class="emoji-row">
+            <div class="face">
+              <img src="grinningFaceBigEyes.png" style="cursor: pointer;" alt="I'm happy";  />
```
Comment (mkimberlin) [inline send-surveys/src/main/resources/emailTemplate.mustache]: Throw in those SVGs on these

Comment (mkimberlin) [reply send-surveys/src/main/resources/emailTemplate.mustache]: Remove the image files as well...

File: send-surveys/build.gradle (hunk 2 above)
Comment (willoci) [reply send-surveys/build.gradle]: done

File: send-surveys/src/main/java/template/mgr/SurveyTemplateManager.java (hunk 1 above)
Comment (willoci) [reply send-surveys/src/main/java/template/mgr/SurveyTemplateManager.java]: done

File: send-surveys/src/main/resources/emailTemplate.mustache (hunk 3 above)
Comment (willoci) [reply send-surveys/src/main/resources/emailTemplate.mustache]: got the img files, the svgs are proving difficult

**Pull request merged**